Assets/HoloToolkit/UX/Scripts/Buttons/ButtonIconProfileTexture.cs

```diff
@@ -0,0 +1,178 @@
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+#if UNITY_WINRT && !UNITY_EDITOR
+#define USE_WINRT
```

Comment:
We should avoid defining our own symbols.

Reply:
Looking into this a bit more has me a bit worried about it's usage. 

https://docs.unity3d.com/Manual/windowsstore-scripts.html

We should probably be using Unity's Win RT enabling definition `#if ENABLE_WINMD_SUPPORT`

Reply:
Just seems like a lot of trouble to get the icon

Reply:
I would definitely agree.  I'll look into it more as it's no my original code.  There are some other bugs this script in particular that are troubling and overly complicated.

thanks,
-pat